### PR TITLE
Added some data

### PR DESCRIPTION
**Pull request by Onkar2991**
@KanchanAnkushe @ShaileshWattamwar @dholeakshay @swapnilr10 @ShraddhaGhogare99 @git4saurabh Review the code and approve as early as possible

### CODE REVIEW

File: src/main/java/com/fileUploadProject/payload/FileResponse.java [hunk 1]
```diff
@@ -6,6 +6,8 @@ public class FileResponse {
    private String message;
    private String fileSize;
 
+   private String fileType;
```
Comment (ShaileshWattamwar) [inline src/main/java/com/fileUploadProject/payload/FileResponse.java:9]: by onkar

**Review by ShaileshWattamwar — APPROVED** [review]
Done

**Review by Onkar2991 — COMMENTED** [review]
ok

**Review by ShraddhaGhogare99 — APPROVED** [review]
Approved by Shraddha

**Review by git4saurabh — APPROVED** [review]
it's fine but do something instead of adding a field

**Review by dholeakshay — APPROVED** [review]

**Review by git4saurabh — APPROVED** [review]
okay

**Review by dholeakshay — APPROVED** [review]
approved

**Review by ShraddhaGhogare99 — APPROVED** [review]

**Review by git4saurabh — APPROVED** [review]
ok akshay

**Review by KanchanAnkushe — APPROVED** [review]

**Pull request merged**